We do not need to check sources, coverage, typing for this project.

--- soupsieve-3.0.3/pyproject.toml.orig
+++ soupsieve-3.0.3/pyproject.toml
@@ -115,10 +115,6 @@
 [tool.hatch.envs.dev]
 dependencies = [
     "pytest",
-    "pytest-cov",
-    "coverage",
-    "mypy",
-    "ruff",
     "build",
     "beautifulsoup4",
     "lxml",
